drift/constants.py

```diff
@@ -0,0 +1,14 @@
+API_VERSION_PREFIX = "/v0"
+APP_URL_PREFIX = "/r/insights/platform/drift"
+AUTH_HEADER_NAME = 'X-RH-IDENTITY'
+FACT_NAMESPACE = "inventory"
```

Comment:
Do we have a card for handling multiple namespaces? I can forsee that being needed.

Reply:
added